packages/manager/src/features/Support/SupportTicketDetail/TicketStatus.tsx

```diff
@@ -42,37 +44,42 @@ export const TicketStatus = (props: Props) => {
   };
 
   return (
-    <Stack
-      sx={(theme) => ({
-        flexFlow: 'row wrap',
-        marginBottom: theme.spacing(3),
-        [theme.breakpoints.down('md')]: {
-          marginLeft: theme.spacing(1),
-        },
-      })}
-    >
-      <Stack
-        sx={{
-          display: 'inline-flex',
-          flexDirection: 'row',
-        }}
-      >
+    <StyledStatusBar container>
+      <Grid alignItems="center" container direction="row" xs>
         <StyledStatusIcon
           ariaLabel={`Ticket status is ${status}`}
           status={status === 'closed' ? 'inactive' : 'active'}
         />
         {capitalize(status)}
-      </Stack>
-      &nbsp;
-      <Typography>
-        | {statusUpdateText} by {updated_by} at {formattedDate}
-      </Typography>
-      &nbsp;
-      {renderEntityLabel()}
-    </Stack>
+        &nbsp;
+        <Typography>
+          | {statusUpdateText} by {updated_by} at {formattedDate}
+        </Typography>
+        &nbsp;
+        {renderEntityLabel()}
+      </Grid>
+      {severity && (
+        <Grid alignItems="center" container direction="row" spacing={1}>
+          <Grid>
+            <Typography>Severity</Typography>
```

Comment:
```suggestion
            <Typography>Severity:</Typography>
```

Should we consider putting a colon after "Severity" here, just to make it clear that the helper text and the chip are related? 

Currently:
![Screenshot 2024-03-26 at 1 44 12 PM](https://github.com/linode/manager/assets/114685994/aedd0787-e166-4015-a0ee-b9930d44b630)

Potentially:
![Screenshot 2024-03-26 at 2 00 09 PM](https://github.com/linode/manager/assets/114685994/87c4991c-9121-4efb-82a0-c516e13d7d15)